requirements/common.in

```diff
@@ -1,7 +1,5 @@
-black~=22.12.0
 click~=8.1.0
-cookiecutter~=2.1.0
-pip-tools~=6.12.0
+cookiecutter~=2.5.0
 pydantic~=1.10.0
-types-python-slugify~=7.0.0
+python-slugify~=8.0.0
 validators~=0.20.0
```

Comment:
```suggestion
validators~=0.22.0
```

Reply:
Unfortunately the latest version breaks stuff. Let's update this requirement in a separate PR.